### ISSUE TITLE
Increase usage of libsecp256k1 by offering a "speed test"

### ISSUE THREAD

**Issue by emergent-reasons** [clifordsymack/Electron-Cash#93]
This is a random idea I had in case low usage of `libsecp256k1` turns out to be a problem and bundling it does not work.

People like speed tests and like to make things faster. So if there is a "speed test" button that would sign N transactions and show the time vs. basically zero for optimized systems, it would show people very concretely how much speed up they can get. A little stamp of approval for optimized systems on the dialog  can be a carrot. A little explanation of blame/ban can be a stick.

**Comment by cculianu** [comment]
A speed test is definitely a UI nicety.  I know I'd like to see it -- it's fun. 

 First we'll try actually packaging it in the linux sdist and/or warning/informing  people in popups and we can do the speed test last -- time permitting.  

I do like speed tests too.

**Comment by emergent-reasons** [comment]
Yes if it can be packaged, that would be much better and this becomes just
for fun.

**Comment by cculianu** [comment]
Ha ha yeah 👍
